### PR TITLE
Remove more legacy code from closor

### PR DESCRIPTION
**Pull request by sharad1126**
Removed old code from closor

#### Status
not tested

#### Description
```DDMcopies``` is not used anymore in campaigns and therefore shouldn't be used in the code as well. 

#### Is it backward compatible (if not, which system it affects?)
no

#### Related PRs
<If it's a follow up work; or porting a fix from a different branch, please mention them here.>

#### External dependencies / deployment changes
campaigns.json and unifiedconfigurations.json

#### Mention people to look at PRs
@z4027163 @vlimant @amaltaro fyi

### CODE REVIEW

**Comment by sharad1126** [comment]
@amaltaro if I am not wrong I think we should also remove the variable ```group_spec ``` https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/closor.py#L572
and it's use in https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/closor.py#L577.

Also, till now the destination for this log was [ ] as it never went into the #L555 so should we also remove the other things from here?

**Comment by amaltaro** [comment]
Yes, I see no reason to keep `group_spec` around, it's always reported as `None`.

**Comment by vlimant** [comment]
what will happen when PPD will ask for the output to go to a specific location?

**Comment by sharad1126** [comment]
@vlimant we have not been using `DDMCopies` parameter since a very long time. The one we use is `toDDM` which is related to unifiedconfiguration.json if I am not wrong and we aren't removing that.

**Comment by sharad1126** [comment]
@amaltaro @vlimant @z4027163 if no one has any more issues, I can merge this.

File: Unified/closor.py [hunk 1]
```diff
@@ -551,30 +551,30 @@ def close(self):
...
+                        #destination_spec = ""
+                        #if destinations:
+                        #    destination_spec = "--destination="+",".join( destinations )
+                        #group_spec = "" ## not used yet 
                         ### should make this a campaign configuration
                         ## inject to DDM when necessary
                         if to_DDM:
                             print "Sending",out," to DDM"
-                            status = pass_to_dynamo( [out], N = n_copies, sites=destinations if destinations else None, group = group_spec if group_spec else None)
+                            status = pass_to_dynamo( [out], N = n_copies, sites=destinations if destinations else None, None)
```
Comment (amaltaro) [inline Unified/closor.py:577]: `destinations` is not initialized inside this if-else block. Either you initialize it as done before `destinations=[]`, or you update this `pass_to_dynamo` line to `sites=None`

Comment (sharad1126) [reply Unified/closor.py:577]: I think there is more here that we are missing.` [L531-L551](https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/closor.py#L531-L551)` is used now but I am not able to see the transfers that should happen for these if `to_DDM` is `true`.

**Comment by haozturk** [comment]
As closor module has changed lately, this PR has some conflicts. I did not check the details, but we will revisit this legacy code in the near future. So, I prefer closing this PR and opening a new one.

**Pull request closed without merging**